### PR TITLE
fix!: Provide primary keys for tables (L)

### PR DESCRIPTION
**Pull request by candiduslynx**
Part of https://github.com/cloudquery/cloudquery/issues/15381

BEGIN_COMMIT_OVERRIDE
fix: Provide primary keys for tables (L) (https://github.com/cloudquery/cloudquery/pull/15469)
BREAKING-CHANGE: Provide primary keys for tables (L) (https://github.com/cloudquery/cloudquery/pull/15469). The following tables are affected:
* `aws_lambda_function_concurrency_configs`
* `aws_lambda_function_event_invoke_configs`
* `aws_lambda_function_event_source_mappings`
* `aws_lambda_layer_version_policies`
* `aws_lightsail_bucket_access_keys`
* `aws_lightsail_container_service_deployments`
* `aws_lightsail_container_service_images`
* `aws_lightsail_database_events`
* `aws_lightsail_database_log_events`
* `aws_lightsail_database_parameters`
* `aws_lightsail_instance_port_states`

END_COMMIT_OVERRIDE

### CODE REVIEW

File: plugins/source/aws/resources/services/lightsail/instance_port_states.go [hunk 1]
```diff
@@ -17,14 +17,17 @@ func instancePortStates() *schema.Table {
 		Name:        tableName,
 		Description: `https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_InstancePortState.html`,
 		Resolver:    fetchLightsailInstancePortStates,
-		Transform:   transformers.TransformWithStruct(&types.InstancePortState{}),
+		Transform: transformers.TransformWithStruct(&types.InstancePortState{},
+			transformers.WithPrimaryKeys("FromPort", "ToPort", "Protocol"),
```
Comment (bbernays) [inline plugins/source/aws/resources/services/lightsail/instance_port_states.go:21]: access to specific ports is based on source addresses so I think we need to include `cidr` and `ipv6cidr` attributes as well

File: plugins/source/aws/docs/tables/aws_lightsail_database_log_events.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ This table depends on [aws_lightsail_databases](aws_lightsail_databases.md).
...
 | ------------- | ------------- |
-|_cq_id (PK)|`uuid`|
+|_cq_id|`uuid`|
 |_cq_parent_id|`uuid`|
 |account_id|`utf8`|
 |region|`utf8`|
-|database_arn|`utf8`|
-|created_at|`timestamp[us, tz=UTC]`|
+|database_arn (PK)|`utf8`|
+|created_at (PK)|`timestamp[us, tz=UTC]`|
```
Comment (bbernays) [inline plugins/source/aws/docs/tables/aws_lightsail_database_log_events.md:22]: I don't think timestamp is unique enough

File: plugins/source/aws/resources/services/lightsail/instance_port_states.go (hunk 1 above)
Comment (candiduslynx) [reply plugins/source/aws/resources/services/lightsail/instance_port_states.go:22]: 4e4858978bf4812ce1f47f41da45907545a26112
inspired by [`aws cli lightsail put-instance-public-ports`](https://docs.aws.amazon.com/cli/latest/reference/lightsail/put-instance-public-ports.html) doc

File: plugins/source/aws/docs/tables/aws_lightsail_database_log_events.md (hunk 2 above)
Comment (candiduslynx) [reply plugins/source/aws/docs/tables/aws_lightsail_database_log_events.md:22]: I was inspired by [`aws cli lightsail get-relational-database-events`](https://docs.aws.amazon.com/cli/latest/reference/lightsail/get-relational-database-events.html) doc here.

I presume, there may be a clashing event for a DB, but IMO it's highly unlikely for now.

We can add the concatenated categories to the PK, though. WDYT?

**Pull request merged**